### PR TITLE
Initial setting of local db in docker-compose

### LINKED ISSUE #11: Initial setting of docker

### PR DESCRIPTION
**Pull request by sonsurim**
# Overview
- Set up the initial setting of local db in docker-compose

Closes #11

### CODE REVIEW

**Review by minsoo-web — COMMENTED** [review]
do we need to use docker-compose?

**Comment by sonsurim** [comment]
@minsoo-web I think docker would be enough, but I set it to docker-compose because it's convenient to manage! Would we use docker?

**Comment by minsoo-web** [comment]
> @minsoo-web I think docker would be enough, but I set it to docker-compose because it's convenient to manage! Would we use docker?

I agree with you on convenience. However, I think you can do it with a dockerfile, what do you think of that?

**Comment by sonsurim** [comment]
@minsoo-web I agree! I think it would be good to discuss it further!

**Comment by sonsurim** [comment]
@minsoo-web As we discussed in discord, I reflected by adding a volume to docker-compose!
[48e37ab](https://github.com/spark-kr/crush-url/pull/12/commits/48e37ab76c983d19a3a200a6adb8ea696f11322a)

File: .gitignore [hunk 1]
```diff
@@ -38,4 +38,7 @@ yarn-error.log*
 *.pem
 
 # Storybook
-**/build-storybook.log
\ No newline at end of file
+**/build-storybook.log
+
+# Docker volumes
+/db-data
```
Comment (minsoo-web) [inline .gitignore:44]: do we need `/` prefix ?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+version: '1'
+
+services:
+  crush-url-db:
+    image: postgres:14.7
+    container_name: crush-url-db
+    ports:
+      - '5434:5432'
```
Comment (minsoo-web) [inline docker-compose.yml:8]:
```suggestion
      - '5432:5432'
```

As far as I know, the default port is 5432, is there any reason to set it to 5434?

Comment (sonsurim) [reply docker-compose.yml:8]: I think we don't need it!

**Comment by sonsurim** [comment]
@minsoo-web I reflected add docker start in [2358425](https://github.com/spark-kr/crush-url/pull/12/commits/2358425a6f8b3e152ca9034d7b0feb31f040b142)

File: docker-compose.yml (hunk 2 above)
Comment (sonsurim) [reply docker-compose.yml]: @minsoo-web I reflected it in [ed9e5b7](https://github.com/spark-kr/crush-url/pull/12/commits/ed9e5b7a01145b15c9eb0a2ae0410c70008bf38d)!

File: .gitignore (hunk 1 above)
Comment (sonsurim) [reply .gitignore]: I reflected it in [660da5f](https://github.com/spark-kr/crush-url/pull/12/commits/660da5f539bff87d5d04bbfc640d1fcc1befb1fd)!

**Review by minsoo-web — APPROVED** [review]
Thanks!

**Pull request merged**